website/content/en/docs/main/tasks/relay/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Relay Cluster
+description: Running basic connectivity between nginx server and client through a relay cluster using ClusterLink.
+---
+
+This task involves creating multi-hop connectivity between a client and a server using relay clusters.
+Multi-hop connectivity using a relay may be necessary for several reasons, such as:
+
+1. When the client and server cannot establish a direct connection due to specifications or regulations.
```

Comment:
If a direct connection is not allowed due to compliance, I would assume that an indirect connection is also not allowed...
Maybe using indirect due to network path limitations would be more reasonable